core/src/main/java/org/apache/metamodel/query/parser/SelectItemParser.java

```diff
@@ -157,6 +161,21 @@ public SelectItem findSelectItem(String expression) throws MultipleSelectItemsPa
             if ("*".equals(columnName)) {
                 throw new MultipleSelectItemsParsedException(fromItem);
             } else if (fromItem.getTable() != null) {
+                if (selectItemHasParenthesis) {
+                    if (functionName.equals(FunctionType.CONCAT.getFunctionName())) {
```

Comment:
The following code should not be CONCAT-specific IMO. It's something that can be used on any set of function parameters.